packages/rancher-vsphere-csi/package.yaml

```diff
@@ -1,3 +1,3 @@
 url: local
 packageVersion: 00
```

Comment:
We will have to consider this as one of our `.000` charts

Reply:
I'm not sure what this means, is there anything that I need to change?

Reply:
No, its fine. Just cc @aiyengar2